### PR TITLE
Initial version of login and registration page and code

### PR DESCRIPTION
**Pull request by shawnlefebre**
Login and registration pages and code to facilitate these functions. Username and password data is stored in localStorage locally on user's browser.

Works in Firefox.  Might have some issues with Chrome.

Currently the registration page does not redirect to the login page.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+/* Below line is used for online Google font */
+@import url(http://fonts.googleapis.com/css?family=Raleway);
+h2{
+    background-color: #FEFFED;
+    padding: 30px 35px;
+    margin: -10px -50px;
+    text-align:center;
+    border-radius: 10px 10px 0 0;
+}
+hr{
```
Comment (petewhartnett) [inline css/style.css:12]: Very cleanly written code, easy to read.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    font-weight: bold;
+    cursor: pointer;
+    width: 100%;
+    border-radius: 5px;
+    padding: 10px 0;
+    outline:none;
+}
+input[type=button]:hover{
+    background: linear-gradient(#ffdd7f 5%, #ffbc00 100%);
+}input[type=submit]{
```
Comment (petewhartnett) [inline css/style.css:81]: "Nitpick" - the input type submit on line 81 can be moved down to 82 for readability, not a big deal though.

File: js/login.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+// Adapted from https://www.formget.com/javascript-login-form/
+
+/*
+loginStorage is a dictionary (Python) where the key is the user name and it contains a dict with:
+ - password: clear text password
+ - TBD
+*/
+
+/**
+ * Log in a Budget App user with their user name and password.  Validates that the
```
Comment (petewhartnett) [inline js/login.js:10]: Great use of comments, it makes this very easy to understand.

**Review by petewhartnett — APPROVED** [review]

File: css/style.css (hunk 2 above)
Comment (shawnlefebre) [reply css/style.css:81]: Completely agree.  For things like this (cosmetic), feel free to change without review.

**Pull request merged**